### PR TITLE
Hosted Gallery AMP page - improvements & tests

### PR DESCRIPTION
**Pull request by lps88**
## What does this change?
- add a hosted gallery page to the amp validation tests
- use appropriate image sizes
- refactoring (extract images to separate template)
- add call to action on last slide
- change fonts to sans serif across all hosted
- add social share buttons

image

image

## Request for comment
@guardian/labs-beta

### CODE REVIEW

**Comment by Calanthe** [comment]
You have to add that test also to the `CommercialAmpValidityTest.scala` file and include fixtures (after running Scala tests in commercial project) like here https://github.com/guardian/frontend/pull/15061/files.

It would be also nice to show some screenshot in the PR's description ;)

**Comment by lps88** [comment]
The page is already released, you can see the screenshots here:
https://github.com/guardian/frontend/pull/15114
There's no visible changes in this PR so seemed unnecessary

**Comment by Calanthe** [comment]
Ahh ok, I was looking for those screenshots at this PR https://github.com/guardian/frontend/pull/15122 ;)

👍 but rememebr to commit the ficture file as well

File: commercial/app/views/hosted/guardianAmpHostedGalleryImage.scala.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@import common.commercial.hosted.HostedGalleryImage
+@import views.support.ImgSrc
+@import common.commercial.hosted.HostedGalleryPage
+
+@(picture: model.ImageMedia, image: HostedGalleryImage, index: Int, page: HostedGalleryPage)
+
+@picture.largestImage.map { largestImage =>
```
Comment (paperboyo) [inline commercial/app/views/hosted/guardianAmpHostedGalleryImage.scala.html]: Will this correctly prioritise Grid’s master assets first (it is the same size as the biggest crop, but better quality, but it is solely the master that should go through the resizer)? CAPI has `isMaster` flag to identify master assets.

Comment (kelvin-chappell) [inline commercial/app/views/hosted/guardianAmpHostedGalleryImage.scala.html]: 'largestImage' is defined here: https://github.com/guardian/frontend/blob/master/common/app/model/Element.scala#L63 and it does take the master image first.

**Pull request merged**